browser/brave_news/BUILD.gn

```diff
@@ -12,6 +12,7 @@ source_set("brave_news") {
     "//brave/components/brave_today/common:common",
     "//brave/components/brave_today/common:mojom",
     "//chrome/browser:browser",
+    "//chrome/browser/favicon:favicon",
```

Comment:
fyi : `"//chrome/browser/favicon"` is sufficient.

Reply:
Nice catch! I cleaned up some other ones too!